econml/cate_estimator.py

```diff
@@ -42,6 +43,14 @@ def _get_inference(self, inference):
         # because inf now stores state from fitting est2
         return deepcopy(inference)
 
+    def _set_input_names(self, Y, T, X):
```

Comment:
Instead of explicitly adding calls to this to `fit` calls in subclasses, could it be added to the logic in `wrap_fit` to happen automatically?

Reply:
Added it to `_prefit`. Still has some issues (e.g. when the ORF overrides the wrapped fit), but I made a workaround for that.